packaging/release/changelogs/changelog.py

```diff
@@ -582,7 +584,7 @@ def combine(fragments):
 
                     result[section] += content
                 else:
-                    result[section] = content
+                    result[section] = [content]
```

Comment:
https://github.com/ansible/ansible/pull/53792 adds type checking for the sections so this won't be needed.

The changelog fragments with the incorrect types were fixed in https://github.com/ansible/ansible/pull/53790